runtime/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -450,42 +446,6 @@ protected boolean supportsNotifications() {
     return true;
   }
 
-  private MetaStoreManagerFactory createMockMetaStoreManagerFactory() {
-    return new MetaStoreManagerFactory() {
-      @Override
-      public PolarisMetaStoreManager getOrCreateMetaStoreManager(RealmContext realmContext) {
-        return metaStoreManager;
-      }
-
-      @Override
-      public Supplier<TransactionalPersistence> getOrCreateSessionSupplier(
-          RealmContext realmContext) {
-        return () -> ((TransactionalPersistence) polarisContext.getMetaStore());
-      }
-
-      @Override
-      public StorageCredentialCache getOrCreateStorageCredentialCache(RealmContext realmContext) {
-        return new StorageCredentialCache(realmContext, configurationStore);
-      }
-
-      @Override
-      public InMemoryEntityCache getOrCreateEntityCache(RealmContext realmContext) {
-        return new InMemoryEntityCache(realmContext, configurationStore, metaStoreManager);
-      }
-
-      @Override
-      public Map<String, PrincipalSecretsResult> bootstrapRealms(
-          Iterable<String> realms, RootCredentialsSet rootCredentialsSet) {
-        throw new NotImplementedException("Bootstrapping realms is not supported");
```

Comment:
was throwing this exception the reason why some tests where using `createMockMetaStoreManagerFactory()` instead of the Mock in the field?

or just left-over code without a particular reason?